### PR TITLE
Fix quantile issue for summaries

### PR DESCRIPTION
**Pull request by wiktork**

### CODE REVIEW

File: src/Microsoft.Diagnostics.Monitoring.WebApi/Metrics/MetricsStore.cs [hunk 1]
```diff
@@ -153,14 +161,17 @@ public async Task SnapshotMetrics(Stream outputStream, CancellationToken token)
         private static string GetMetricLabels(ICounterPayload metric, double? quantile)
         {
             string metadata = metric.Metadata;
+
+            char separator = IsMeter(metric) ? '=' : ':';
+            var metadataValues = CounterUtilities.GetMetadata(metadata, separator);
             if (quantile.HasValue)
             {
-                metadata = CounterUtilities.AppendPercentile(metadata, quantile.Value);
+                metadataValues.Add("quantile", quantile.Value.ToString(CultureInfo.InvariantCulture));
```
Comment (kkeirstead) [inline src/Microsoft.Diagnostics.Monitoring.WebApi/Metrics/MetricsStore.cs:169]: In converting from percentile to quantile, do we need to divide by 100? E.g. 95% is 0.95 quantile I think? Or is that already getting taken care of

Comment (wiktork) [reply src/Microsoft.Diagnostics.Monitoring.WebApi/Metrics/MetricsStore.cs:169]: quantiles all start as values between 0 and 1. AppendPercentile converts them to percentages.

File: src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/MetricsFormattingTests.cs [hunk 2]
```diff
@@ -54,9 +53,9 @@ public async Task HistogramFormat_Test()
             // should we call this method, or should this also be implicitly testing its behavior by having this hard-coded?
             string metricName = $"{MeterName.ToLowerInvariant()}_{payload[0].Name}";
 
-            const string percentile_50 = "{Percentile=\"50\"}";
-            const string percentile_95 = "{Percentile=\"95\"}";
-            const string percentile_99 = "{Percentile=\"99\"}";
+            const string percentile_50 = "{quantile=\"0.5\"}";
```
Comment (kkeirstead) [inline src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/MetricsFormattingTests.cs:56]: nit: refactor these to `quantile_xx`

**Review by kkeirstead — APPROVED** [review]

**Review by kkeirstead — APPROVED** [review]

**Pull request merged**